### ISSUE TITLE
den shell doesn't work

### ISSUE THREAD

**Issue by ihor-sviziev** [swiftotter/den#61]
Looks like we have some issue with bash. It just shows empty string:
```
➜  myproj git:(main) den shell
➜  myproj git:(main)
➜  myproj git:(main) den env exec php-fpm bash
➜  myproj git:(main)
➜  myproj git:(main) den env exec php-fpm sh
/var/www/html $
```

**Comment by navarr** [comment]
Which version of PHP are you using here?

**Comment by ihor-sviziev** [comment]
7.4

**Comment by ihor-sviziev** [comment]
@navarr I think the issue coming from https://github.com/swiftotter/den/commit/ab972dec25db331e05a973186e7baa7f4ef30643#diff-d674543fbfb99b6b65ef458b2f1b00fdddf64d8c56fc997eb74547f798d8048dR2

**Comment by navarr** [comment]
@ihor-sviziev You are correct.  I'll have to update my testing procedures.  Images are building now to fix.  Workaround is to launch `sh` instead of bash and modify that file until the new images are available
